<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="snapShotButton">
      <property name="text">
       <string>Snapshot</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTransform_Matrix">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionShow_Dialog"/>
    <addaction name="actionSave_Map"/>
    <addaction name="actionLoad_Map"/>
    <addaction name="actionLoad_PCDs"/>
   </widget>
   <addaction name="menuTransform_Matrix"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionX_translation">
   <property name="text">
    <string>X Translation</string>
   </property>
  </action>
  <action name="actionY_Translation">
   <property name="text">
    <string>Y Translation</string>
   </property>
  </action>
  <action name="actionZ_Translation">
   <property name="text">
    <string>Z Translation</string>
   </property>
  </action>
  <action name="actionX_Rotation">
   <property name="text">
    <string>X Rotation</string>
   </property>
  </action>
  <action name="actionY_Rotation">
   <property name="text">
    <string>Y Rotation</string>
   </property>
  </action>
  <action name="actionZ_Rotation">
   <property name="text">
    <string>Z Rotation</string>
   </property>
  </action>
  <action name="actionShow_Dialog">
   <property name="text">
    <string>Transform Matrix</string>
   </property>
  </action>
  <action name="actionSave_Map">
   <property name="text">
    <string>Save Map</string>
   </property>
  </action>
  <action name="actionLoad_Map">
   <property name="text">
    <string>Load Map</string>
   </property>
  </action>
  <action name="actionLoad_PCDs">
   <property name="text">
    <string>Load PCDs</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
